distribution/docs/src/main/resources/_contents/_tables/ddf.catalog.federation.impl.CachingFederationStrategy-table-contents.adoc

```diff
@@ -0,0 +1,69 @@
+.[[ddf.catalog.federation.impl.CachingFederationStrategy]]Catalog Federation Strategy
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Maximum start index
+|maxStartIndex
+|Integer
+|Sets a limit on the number of results this sorted federation strategy can handle from each federated source. A large start index         in conjunction with several federated sources could yield a large result set, which the sorted federation strategy has a limited ability to do.           The admin can make a rough calculation to decide what maximum start index to use based on the amount of memory in the system, the amount of          federated sources, the number of threads, and the expected amount of query results requested          ( (average # of threads) * (maximum # of federated sources) * (maxStartIndex + maximumQueryResults) ) must fit into the allocated memory of          the running distribution.  This field will be removed when sorted federation strategy has the ability to sort a larger amount of results.
```

Comment:
Extra spacing in this description
![image](https://cloud.githubusercontent.com/assets/8041246/18289505/846d79a4-7434-11e6-977a-5b4a2993389b.png)